packages/devtools_app/lib/src/shared/ui/file_import.dart

```diff
@@ -288,7 +288,7 @@ Future<List<XFile>> importRawFilesFromPicker({
 
 @visibleForTesting
 Future<DevToolsJsonFile> toDevToolsFile(XFile file) async {
-  final data = jsonDecode(await file.readAsString());
+  final data = json.fuse(utf8).decode(await file.readAsBytes())!;
```

Comment:
This will give us perf improvements everywhere where this helper is used. Nice win!

files can also be imported via drag and drop. Are there similar improvements that can be made here: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/config_specific/drag_and_drop/_drag_and_drop_web.dart/#L83

Reply:
Ah yeah, I will add it here to, I can just search the whole codebase.

Reply:
done, many of these the performance probably doesn't matter but it doesn't hurt

Reply:
Actually, in discussion with the JS interop folks I removed this change because the performance will need to be separately evaluated for wasm/dart2js, it isn't a given that this will be better in an interop context.